src/.preload.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
This file is not needed. I don't know where it came from, but I occasionally see it popping up on my end as well. Regardless, we can remove it here.